roles/aws.ec2-autoscaling-group/tasks/main.yml

```diff
@@ -65,6 +72,7 @@
       user_data: "{{ _launch_config_user_data }}"
       instance_profile_name: "{{ asg_launch_config_instance_profile_name }}"
       assign_public_ip: "{{ asg_launch_config_assign_public_ip }}"
+      volumes: "{{ _launch_config_volumes }}"
```

Comment:
I know it's out of scope for this PR, but can you please include "{{ _launch_config_volumes + asg_additional_volumes }}, with default being [] ?